docs/source/daml/troubleshooting.rst

```diff
@@ -185,7 +185,7 @@ Now that the contract is bound to a variable, we can check whether it has some e
 
 .. _faqs-must-fail:
 
-How to test that an update action cannot be committed
+How To Test That an Update Action Cannot Be Committed
```

Comment:
Is "to" supposed to be capitalized or not? I see both.

Reply:
I checked this on a title case converter tool using AP style as the default, the results include To being capitalized (but not "In" above, I will change that).